server/container_create_linux.go

```diff
@@ -867,14 +769,18 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 
 	sort.Sort(orderedMounts(mounts))
 
+	rspecMount := rspec.Mount{
+		Destination: "",
```

Comment:
No need to initialize fields with default values (for string the default value is "").

Reply:
fixed!